tests/unit/modules/test_cluster_role_info.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+import pytest
+
+from ansible_collections.sensu.sensu_go.plugins.module_utils import (
+    errors, utils,
+)
+from ansible_collections.sensu.sensu_go.plugins.modules import cluster_role_info
+
+from .common.utils import (
+    AnsibleExitJson, AnsibleFailJson, ModuleTestCase, set_module_args,
+)
+
+
+class TestClusterRoleInfo(ModuleTestCase):
+    def test_get_all_roles(self, mocker):
+        get_mock = mocker.patch.object(utils, "get")
+        get_mock.return_value = [1, 2, 3]
+        set_module_args()
+
+        with pytest.raises(AnsibleExitJson) as context:
+            cluster_role_info.main()
+
+        _client, path = get_mock.call_args[0]
+        assert path == "/clusterroles"
+        assert context.value.args[0]["objects"] == [1, 2, 3]
+
+    def test_get_single_role(self, mocker):
```

Comment:
Same as above, I suggest renaming to `test_get_single_cluster_role` for consistency.